statsmodels/stats/stattools.py

```diff
@@ -100,8 +100,8 @@ def jarque_bera(resids, axis=0):
     """
     resids = np.asarray(resids)
     # Calculate residual skewness and kurtosis
-    skew = stats.skew(resids, axis=axis)
-    kurtosis = 3 + stats.kurtosis(resids, axis=axis)
+    skew = np.array(stats.skew(resids, axis=axis), ndmin=axis)
```

Comment:
``np.asarray`` would be better, to not make a copy for older scipy releases (and future ones, where this issue should be fixed).